### ISSUE TITLE
Problem with YGG and Scenetime ... keep looping on Try to find the Cloudflare 'Verify you are human' button

### ISSUE THREAD

**Issue by ollie2a** [FlareSolverr/FlareSolverr#1014]
### Have you checked our README?

- [X] I have checked the README

### Have you followed our Troubleshooting?

- [X] I have followed your Troubleshooting

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you checked the discussions?

- [X] I have read the Discussions

### Environment

```markdown
- FlareSolverr version: 3.3.11
- Last working FlareSolverr version: ?
- Operating system: Linux
- Are you using Docker: yes
- FlareSolverr User-Agent (see log traces or / endpoint): Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36
- Are you using a VPN: no
- Are you using a Proxy: no
- Are you using Captcha Solver: yes
- If using captcha solver, which one: hcaptcha-solver
- URL to test this issue: https://www3.yggtorrent.qa/engine/search?do=search&order=desc&sort=publish_date&category=all
```

### Description

I am using Jackett and log in debug and also html dump - seems it loops on this and do not get pass the initial page ..  same issue with other torrents. 

These are my environment variables : 

LOG_LEVEL=debug
HOSTNAME=flaresolverr
PYTHON_VERSION=3.11.7
PWD=/app
PYTHON_SETUPTOOLS_VERSION=65.5.1
TZ=Europe/London
HOME=/app
LANG=en_GB
CAPTCHA_SOLVER=hcaptcha-solver
GPG_KEY=A035C8C19219BA821ECEA86B64E628F8D684696D
TERM=xterm
SHLVL=1
PYTHON_PIP_VERSION=23.2.1
PYTHON_GET_PIP_SHA256=9cc01665956d22b3bf057ae8287b035827bfd895da235bcea200ab3b811790b6
PYTHON_GET_PIP_URL=https://github.com/pypa/get-pip/raw/4cfa4081d27285bda1220a62a5ebf5b4bd749cdb/public/get-pip.py
PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
LOG_HTML=false

And the HTML file from Cloudflare ... looks like it keep spinning without moving to the site or challenge ... 

[untitled.txt](https://github.com/FlareSolverr/FlareSolverr/files/13693285/untitled.txt)

Unfortunately I am headless so can't see what is happening on the site. Is there a way to connect to docker with VNC or similar to see what the browser is doing ? 

Cheers

### Logged Error Messages

```text
2023-12-16 17:09:02 INFO     ReqId 140627137328896 Incoming request => POST /v1 body: {'maxTimeout': 55000, 'cmd': 'request.get', 'url': 'https://www3.yggtorrent.qa/engine/search?do=search&order=desc&sort=publish_date&category=all'}
2023-12-16 17:09:02 DEBUG    ReqId 140627137328896 Launching web browser...
version_main cannot be converted to an integer
2023-12-16 17:09:02 DEBUG    ReqId 140627137328896 Started executable: `/app/chromedriver` in a child process with pid: 170
2023-12-16 17:09:03 DEBUG    ReqId 140627137328896 New instance of webdriver has been created to perform the request
2023-12-16 17:09:03 DEBUG    ReqId 140627103758080 Navigating to... https://www3.yggtorrent.qa/engine/search?do=search&order=desc&sort=publish_date&category=all
2023-12-16 17:09:05 INFO     ReqId 140627103758080 Challenge detected. Title found: Just a moment...
2023-12-16 17:09:05 DEBUG    ReqId 140627103758080 Waiting for title (attempt 1): Just a moment...
2023-12-16 17:09:06 DEBUG    ReqId 140627103758080 Timeout waiting for selector
2023-12-16 17:09:06 DEBUG    ReqId 140627103758080 Try to find the Cloudflare verify checkbox...
2023-12-16 17:09:06 DEBUG    ReqId 140627103758080 Cloudflare verify checkbox not found on the page.
2023-12-16 17:09:06 DEBUG    ReqId 140627103758080 Try to find the Cloudflare 'Verify you are human' button...
2023-12-16 17:09:06 DEBUG    ReqId 140627103758080 The Cloudflare 'Verify you are human' button not found on the page.
2023-12-16 17:09:08 DEBUG    ReqId 140627103758080 Waiting for title (attempt 2): Just a moment...
2023-12-16 17:09:09 DEBUG    ReqId 140627103758080 Timeout waiting for selector
2023-12-16 17:09:09 DEBUG    ReqId 140627103758080 Try to find the Cloudflare verify checkbox...
2023-12-16 17:09:10 DEBUG    ReqId 140627103758080 Cloudflare verify checkbox found and clicked!
2023-12-16 17:09:10 DEBUG    ReqId 140627103758080 Try to find the Cloudflare 'Verify you are human' button...
```

**Comment by ollie2a** [comment]
Awwww sorry fixed with the latest version v3.3.12  ... did not see this one was published

**Comment by ollie2a** [comment]
fixed in v3.3.12
